configuration/api.go

```diff
@@ -511,13 +492,13 @@ func (a *API) UpdateTag(name string, groupIDs []int) error {
 }
 
 // Lists properties of groups
-func (a *API) ListGroupsProperties(namespace string, namePrefix string, groupIDs []int) ([]GroupProperties, error) {
+func (a *API) ListGroupsProperties(groupIDs []int, opts *ListGroupsPropertiesOptions) ([]GroupProperties, error) {
```

Comment:
Namespace and namePrefix are also optional in `ListLicenseProperties`, please update it there in the same way.

Also there is `namespace_prefix` instead of `namespace` in `ListLicenseProperties` for versions v3.3+, please fix it too

Reply:
then we have a bug in our docs https://developers.livechat.com/docs/management/configuration-api/v3.6#list-groups-properties

Reply:
> Also there is namespace_prefix instead of namespace in ListLicenseProperties for versions v3.3+, please fix it too

Can you link where it is? I've checked both external docs and code and it looks like it's `namespace` https://github.com/livechat/api/blob/master/services/configuration-api/parsers/v3.6/methods/list_groups_properties.go#L28

Reply:
Ok, now I get what you mean, there is a mistake in SDK that needs to be fixed